lib/components/artist/shows/small_list.js

```diff
@@ -61,16 +61,14 @@ const styles = StyleSheet.create({
 
 export default Relay.createContainer(SmallList, {
   fragments: {
-    artist: () => Relay.QL`
-      fragment on Artist {
-        partner_shows(status: "closed") {
-          meta_image {
-            cropped(width: 75, height: 75) {
-              url
-            }
+    shows: () => Relay.QL`
```

Comment:
Well, I’m glad you asked. I thought it was just a Relay transpolar thing, but it turns out to be a ES6 string literal that allows interpolation 🎉 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals